README.md

```diff
@@ -1,34 +1,131 @@
 # smart-gateway-operator
 
-Operator for the smart gateway available
-https://github.com/redhat-service-assurance/smart-gateway
+Operator for the infra.watch [smart gateway](https://github.com/redhat-service-assurance/smart-gateway)
 
-# Building
+## Deployment
```

Comment:
Could this section somehow be broken out so that it's more obvious this is assumed to be executed against a running cluster? The sections below it are almost a separate grouping. Maybe a separate file, or just a section break, or something to make it feel more independent of the testing sections below?

Reply:
I'll expand the headings a bit to make them more clear